### PR TITLE
build(demo): set GitHub branch to V4.x

### PR DESCRIPTION
**Pull request by paodb**

### CODE REVIEW

**Review by javier-godoy — CHANGES_REQUESTED** [review]
Commit must be based on 4.x branch

**Review by javier-godoy — APPROVED** [review]

File: src/test/java/com/flowingcode/vaadin/addons/orgchart/OrgchartDemoView.java [hunk 1]
```diff
@@ -32,6 +33,7 @@
 @StyleSheet("context://frontend/styles/orgchart/demo-styles.css")
 @StyleSheet("context://frontend/styles/orgchart/font-awesome.css")
 @GithubLink("https://github.com/FlowingCode/OrgChartAddon")
+@GithubBranch("4.x")
```
Comment (javier-godoy) [inline src/test/java/com/flowingcode/vaadin/addons/orgchart/OrgchartDemoView.java]: Branch is `v4x`, not `4.x`

**Review by javier-godoy — CHANGES_REQUESTED** [review]
Branch is `v4.x`, not `4.x`

File: src/test/java/com/flowingcode/vaadin/addons/orgchart/OrgchartDemoView.java (hunk 1 above)
Comment (paodb) [reply src/test/java/com/flowingcode/vaadin/addons/orgchart/OrgchartDemoView.java]: Fixed

**Review by javier-godoy — APPROVED** [review]

**Pull request merged**